src/SmartComponents/DriftPage/DriftPage.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { Main, PageHeader, PageHeaderTitle } from '@red-hat-insights/insights-frontend-components';
+import { Card, CardBody } from '@patternfly/react-core';
+
+import { DriftTable } from './DriftTable/DriftTable';
+
+export class DriftPage extends Component {
```

Comment:
Try switching this to export default

Reply:
It was 2 issues. Needed to:
```
export default DriftPage;
```
and needed to:
```
import DriftTable from './DriftTable/DriftTable';
```